server/src/main/scala/org/http4s/server/ServerBuilder.scala

```diff
@@ -68,14 +69,26 @@ trait SSLSupport { this: ServerBuilder =>
               protocol: String = "TLS",
             trustStore: Option[StoreInfo] = None,
             clientAuth: Boolean = false): Self
+
+}
+
+trait SSLContextSupport { this: ServerBuilder =>
+  def withSSLContext(sslContext: SSLContext, clientAuth: Boolean = false): Self
 }
+
+sealed trait SSLBits
```

Comment:
If we came up with a better name (`SslConfig`?), we could leave `SslBits` as a deprecated alias and keep our migration notes in the compiler.